File: WebFt30/src/app/features/rfts-setup/components/platform-management/user-accounts/components/user-edit-dialog/application-user-patch.ts
export class ApplicationUserPatch {
  userName!: string; // required to find existing user
  firstName: string | undefined = undefined;
  lastName: string | undefined = undefined;
  role: string | undefined = undefined;
  jobTitle: string | undefined = undefined;
  email: string | undefined = undefined;
  phoneNumber: string | undefined = undefined;
  password: string | undefined = undefined;
}

File: Backend/src/Fibertest30.Application/Identity/CQRS/Commands/UpdateUser/ApplicationUserPatch.cs
namespace Fibertest30.Application;

// Do not change the names of properties as they are used in reflection.
public record ApplicationUserPatch
(
    string? UserName,
    string? FirstName,
    string? LastName,
    string? Email,
    string? PhoneNumber,
    string? JobTitle,
    string? Role,
    string? Password
    );
